src/components/ArtistName.css

```diff
@@ -0,0 +1,29 @@
+.artist-name-link:not(:first-of-type)::before {
+  content: " , ";
```

Comment:
This should be `", ";` (without a space between the opening `"` and the `,`)

Reply:
Fixed!